src/plugins/tables/tables.js

```diff
@@ -173,7 +174,12 @@ $document.on( "draw.dt", selector, function( event, settings ) {
 	// Identify that the table has been updated
 	$elm.trigger( "wb-updated" + selector, [ settings ] );
 } );
-
+$document.on( "keypress", ".paginate_button", function( event ) {
```

Comment:
Be more specific for the CSS event selector.

Reply:
Done. Here is a new solution

// This is required to override the datatable.js (v1.10.13) behavior to cancel the event propagation on anchor element.
$elm.next( ".bottom" ).find( ".paginate_button" )
			.attr( {
				"href": "#" + $elm.context.id
			} )			
.on( "keypress", function( evn ){
				if ( evn.keyCode === 13 ){
					window.location = evn.target.href;
				}
			} )